age-app/src/app/filling-container.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+import React from "react";
+import { useState } from "react";
+import "./filling-container.css";
+import FillingBox from "./filling-box";
+export default function FillingContainer() {
+  //store input day, month, year
+  const [day, setDay] = useState("");
+  const [month, setMonth] = useState("");
+  const [year, setYear] = useState("");
+//store the calculation of days, month, year
+  const [resultDay, setResultDay] = useState("--");
```

Comment:
You should not initialize it to be "--", that is for presentation not the actual data.